platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -208,6 +282,32 @@ - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshot
     });
 }
 
+#if TARGET_OS_IPHONE
+- (UIView *)attributionBackgroundView:(CGRect)frame
+{
+    UIView *attributionBackgroundView = [[UIView alloc] initWithFrame:frame];
+    attributionBackgroundView.layer.backgroundColor = [UIColor grayColor].CGColor;
+    attributionBackgroundView.layer.cornerRadius = 4;
+    attributionBackgroundView.layer.opacity = 0.6;
+    
+    return attributionBackgroundView;
+}
+#endif
+
+- (void)drawAttributionText:(NSArray *)attributionInfo origin:(CGPoint)origin
+{
+    int x = origin.x;
+    int y = origin.y;
```

Comment:
As general code hygiene, use `CGFloat` for individual x and y values, or preferably combine them into a `CGPoint`.